src/objects/index.ts

```diff
@@ -39,23 +40,29 @@ export function start({ stage, props }: StartParameters): void {
   startEdgeCulling()
   startSettings()
   startSelection()
+  startCache()
 }
 
 export function stop(): void {
   emitter.clear()
 
-  stopApplication()
-  stopViewport()
-  stopScale()
-  stopGuides()
-  stopStage()
-  stopNodes()
-  stopConfig()
-  stopScope()
-  stopFonts()
-  stopCulling()
-  stopLabelCulling()
-  stopEdgeCulling()
-  stopSettings()
-  stopSelection()
+  try {
+    stopApplication()
+    stopViewport()
+    stopScale()
+    stopGuides()
+    stopStage()
+    stopNodes()
+    stopConfig()
+    stopScope()
+    stopFonts()
+    stopCulling()
+    stopLabelCulling()
+    stopEdgeCulling()
+    stopSettings()
+    stopSelection()
+    stopCache()
+  } catch (error) {
+    console.error(error)
+  }
```

Comment:
Added this while debugging the viewport teardown issue. Even if something errors this will at least prevent the router from breaking.